karaf/apache-brooklyn/src/main/resources/etc/org.apache.brooklyn.classrename.cfg

```diff
@@ -0,0 +1,21 @@
+################################################################################
```

Comment:
@m4rkmckenna Do you mean that I shouldn't add it manually there but just add a feature on the `feature.xml`? Is it a pattern well known? It seems we currently do not use that